repose-aggregator/src/docs/asciidoc/filters/api-validator.adoc

```diff
@@ -11,45 +26,283 @@ include::../_includes/in-progress.adoc[]
 
 == Prerequisites & Postconditions
 === Required headers
-???.
+This filter does not require any request headers.
+
+However, the configuration can require specific headers and/or values be present in order to successfully exit this filter.
+The most common use of a request header by this filter is for <<../recipes/role-based-access-control.adoc#,Role-Based Access Control (RBAC)>> using the `X-Roles` header populated by the <<../filters/keystone-v2.adoc#,Keystone v2 filter>>.
 
 === Required preceding filters
-???.
+This filter has no dependencies on other filters and can be placed wherever it is needed in the filter chain.
+
+However, due to the nature of this filter it is typically placed early in the filter chain immediately after any authentication filters.
 
 === Request headers created
-???.
+This filter only creates/modifies request headers when configured for <<../recipes/delegation.adoc#,Delegation>>.
+If delegation is enabled, then the `X-Delegated` header is created.
+This is mainly intended for use by the <<herp.adoc#, Highly Efficient Record Processor (HERP) filter>> and <<derp.adoc#, Delegation Response Processor (DeRP) filter>> for internal delegation processing within Repose.
+However, it can be exposed to the origin service under certain configurations.
 
 === Request body changes
-???.
+This filter does not modify the response body.
 
 === Recommended follow-on (succeeding) filters
-???.
+This filter is not strictly required by any other filters.
 
 === Response body changes
-???.
+This filter does not modify the response body.
 
 === Response headers created
-???.
+This filter does not create/modify any response headers.
 
 === Response Status Codes
```

Comment:
Alright then. I'm sure we will hit this inconsistency at some point in the future, and normalize all section headers. Just figured I may as well call attention to it again.